BidCat/API/BiddingController.cs

```diff
@@ -14,22 +14,80 @@ public class BiddingController : ApiProvider
 		public override IEnumerable<ApiCommand> Commands => new List<ApiCommand>
 		{
 			//The Auction ID must be first in the URL when it's needed, same for the Bank ID
-			new ApiCommand("RegisterBank", args => args.Count() == 3 ? CreateBank(args.ToArray()[0], args.ToArray()[1], args.ToArray()[2]) : throw new ApiError("Incorrect number of argument, expected 3"), new List<ApiParameter>{ new ApiParameter("Users Collection Name", "string"), new ApiParameter("Transactions Collection Name", "string"), new ApiParameter("Currency Field Name", "string") }, "Registers a bank system. Returns the ID of the bank."),
+			new ApiCommand("RegisterBank", args => args.Count() == 3 ? CreateBank(args.ToArray()[0], args.ToArray()[1], args.ToArray()[2]) : args.Count() == 1 ? CreateBank(JsonConvert.DeserializeObject<List<string>>(args.First())) : throw new ApiError("Incorrect number of argument, expected 3"), new List<ApiParameter>{ new ApiParameter("Users Collection Name", "string"), new ApiParameter("Transactions Collection Name", "string"), new ApiParameter("Currency Field Name", "string") }, "Registers a bank system. Returns the ID of the bank."),
 			new ApiCommand("RegisterAuction", args => args.Any() ? RegisterAuction(int.Parse(args.First())) : throw new ApiError("Bank ID was not specified"), new List<ApiParameter> { new ApiParameter("Bank ID") }, "Registers a new auction."),
-			new ApiCommand("GetReservedMoney", args => GetAuction(args.Any() ? int.Parse(args.First()) : throw new ApiError("Auction ID was not specified")).GetReservedMoney(args.Skip(1).Count() == 1 ? JsonConvert.DeserializeObject<IEnumerable<int>>(args.Skip(1).First()) : args.Skip(1).Any() ? args.Skip(1).Select(int.Parse) : throw new ApiError("User ID was not specified")).Wait(), new List<ApiParameter>{ new ApiParameter("Auction ID"), new ApiParameter("User Id") }, "Finds the total reserved money for a given user."),
+			new ApiCommand("GetReservedMoneyAuction", args => GetAuction(args.Any() ? int.Parse(args.First()) : throw new ApiError("Auction ID was not specified")).GetReservedMoney(args.Skip(1).Count() == 1 ? JsonConvert.DeserializeObject<IEnumerable<int>>(args.Skip(1).First()) : args.Skip(1).Any() ? args.Skip(1).Select(int.Parse) : throw new ApiError("User ID was not specified")).Wait(), new List<ApiParameter>{ new ApiParameter("Auction ID"), new ApiParameter("User Id") }, "Finds the total reserved money for a given user."),
```

Comment:
This entire file has lots of copypaste.

If API commands are going to share parameters, declare the parameters and their validators elsewhere in the class and reference them here. 

See [tpp-BizHawk2's Memory API](https://github.com/TwitchPlaysPokemon/tpp-BizHawk2/blob/master/BizHawk.Client.Common/Api/Public/Providers/MemoryApi.cs) for examples of how I broke out validators and parameters I intended to reuse.